examples/tensorflow/segmentation/evaluation.py

```diff
@@ -46,9 +47,9 @@ def get_argument_parser():
         '--mode',
         '-m',
         nargs='+',
-        choices=['train', 'test', 'export'],
+        choices=['test', 'export'],
```

Comment:
What reason for these changes?

Reply:
It's as you've told me, since `train.py` only supports "train" mode, I suppose that `evaluation.py` should only support the other two.

Reply:
`evaluation.py` can run in the training mode. In this case, it will listen `train.py` and evaluate checkpoints that were saved by `train.py`.

Reply:
Made `evaluation.py` use common parser with `mode=True` so that it has all three CLI options for `-m`